### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by melinath**
Note: the release branch reverts https://github.com/hashicorp/terraform-provider-google-beta/pull/3022 due to it breaking a test case. Build to verify the test now passes: https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloudBeta_ProviderGoogleCloudBetaGoogleProject/176822

### CODE REVIEW

**Comment by melinath** [comment]
attn @c2thorn re: the revert here since the fix _should_ be in next release. Tracking ticket is here: https://github.com/hashicorp/terraform-provider-google/issues/8667

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+* compute: added `mtu` field to `google_compute_interconnect_attachment` ([#3006](https://github.com/hashicorp/terraform-provider-google-beta/pull/3006))
+* compute: added autoscaling_policy.cpu_utilization.predictive_method field to `google_compute_autoscaler` and `google_compute_region_autoscaler` ([#2987](https://github.com/hashicorp/terraform-provider-google-beta/pull/2987))
+* compute: added support for `nic_type` to `google_compute_instance` (GA only) ([#2998](https://github.com/hashicorp/terraform-provider-google-beta/pull/2998))
+* compute: fixed perma-diff for cos-family disk images ([#3024](https://github.com/hashicorp/terraform-provider-google-beta/pull/3024))
+* compute: Fixed service account scope alias to be updated. ([#3021](https://github.com/hashicorp/terraform-provider-google-beta/pull/3021))
+* container: fixed container cluster not removed from the state when received 404 error on delete call for the resource `google_container_cluster` ([#3018](https://github.com/hashicorp/terraform-provider-google-beta/pull/3018))
```
Comment (c2thorn) [inline CHANGELOG.md:16]: I believe these are considered bug fixes (their tracking issue was a bug)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+* container: added field `ephemeral_storage_config` to resource `google_container_node_pool` and `google_container_cluster` (beta) ([#3023](https://github.com/hashicorp/terraform-provider-google-beta/pull/3023))
+* datafusion : new instance type added for the resource `google_data_fusion_instance` ([#3015](https://github.com/hashicorp/terraform-provider-google-beta/pull/3015))
```
Comment (c2thorn) [inline CHANGELOG.md:18]:
```suggestion
* datafusion : added support for the `DEVELOPER` instance type to `google_data_fusion_instance` ([#3015](https://github.com/hashicorp/terraform-provider-google-beta/pull/3015))
```

**Review by c2thorn — CHANGES_REQUESTED** [review]

**Comment by melinath** [comment]
@c2thorn thanks! Fixed.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**